TheOtherRoles/TheOtherRoles.cs

```diff
@@ -159,19 +159,34 @@ public static class Engineer {
             public static Color color = new Color32(0, 40, 245, byte.MaxValue);
             private static Sprite buttonSprite;
 
-            public static int remainingFixes = 1;           
+            public static int remainingFixes = 1;
+            public static int engineerNumberOfFixes = 1;
             public static bool highlightForImpostors = true;
-            public static bool highlightForTeamJackal = true; 
-
+            public static bool highlightForTeamJackal = true;
+            public static bool resetFixesAfterMeeting = false;
             public static Sprite getButtonSprite() {
                 if (buttonSprite) return buttonSprite;
                 buttonSprite = Helpers.loadSpriteFromResources("TheOtherRoles.Resources.RepairButton.png", 115f);
                 return buttonSprite;
             }
 
+            private static bool createResetFixesAfterMeeting(int engineerFixesPer)
+            {
+                //0 for per round
+                //1 for per game
+                return engineerFixesPer == 0;
```

Comment:
The magic strings "Round" and "Game" should ideally be in an enum, you could name it something like ResetOptions.
Then you can compare to an easier to understand expression like this:
`return engineerFixesPer == (int) ResetOptions.Round;`

That avoids the need to have inline comments explaining the enum values.